test/uk/gov/hmrc/voapropertylinking/connectors/modernised/ExternalOrganisationManagementApiSpec.scala

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.voapropertylinking.connectors.modernised
+
+import basespecs.BaseUnitSpec
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.when
+import uk.gov.hmrc.voapropertylinking.models.modernised.agentrepresentation.{AgentDetails, AppointmentAction, AppointmentChangeResponse, AppointmentChangesRequest, AppointmentScope}
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+class ExternalOrganisationManagementApiSpec extends BaseUnitSpec {
+
```

Comment:
I don't think these tests have much value. What is useful in connector tests is to ensure that the correct query parameters are included in the http call. That is missing here.

Reply:
did you consider this?

Reply:
this.